### ISSUE TITLE
Kitchen Automats

### ISSUE THREAD

**Issue by Midaychi** [d3athrow/vgstation13#3198]
A feature request: Replacing those hallway side kitchen table with a few automats.

Tl;Dr imagine doing this while a clown sneaks up behind you in the medbay-science hallway:

http://media2.s-nbcnews.com/j/msnbc/Components/Photos/060828/060828_automat_bcol_2p.grid-6x2.jpg

Automat: Dense vending machines that start empty and are difficult to see through. Can be unlocked and programmed with chef id or better, but only from the kitchen side.

http://newyorkdailyphoto.com/nydppress/wp-content/uploads/2006/10/AutomatRedux11.jpg

Chef can place 'edible' objects in automat and define whether they are free or cost things. Costs are defined entirely by the chef and proceeds go directly to the linked id's account (possibly after nt tax).

Automat can be loaded but only from the kitchen side.

Automat changes sprite between empty sprite and containing things sprite based on if it contains vendable objects or not.

http://www.photographersgallery.com/i/full/automat_ny.jpg

If chef still wants to lay shit out for people to consume, they can do it on the bar kitchen lobby side.

**Comment by CptWad** [comment]
I think it would be cool. But only like half the kitchen counter as some chefs would want a regular table plus table means you can get over/through it

**Comment by Midaychi** [comment]
I spose the end result is up to mappers; I really just want these automats.

**Comment by ComicIronic** [comment]
The Automat is actually a really cool idea.
It would also be useful in implementing more economy for other departments, since you wouldn't need to be at the desk to sell items.

**Comment by Midaychi** [comment]
Not sure how you'd do it, but being able to set the space-automats into barter mode would be hilarious.
Also maybe stupid and over complicated? I dunno. Items and material goods seem a lot more useful in ss13 than money.

Example: Cargotech puts a reflective armor vest into cargo automat, but instead of setting a price they set it to 'barter', and define what the automat would accept in payment. 
You could choose a category, like "armors, clothes, coins, resources, id cards, weapons" and would have a slider for 'quality'. The lower the quality, the less picky the automat is (So if it were set to coins, it would accept iron coins if it was set all the way low) but the higher the quality is, the more picky the automat is (Putting it up to 75% quality and setting it to gloves might make it only accept insulated gloves and cranking it up past that might only accept fancy hard to get gloves)

Someone interested in buying an object could see what category and quality the item for barter is set to, and can insert items into the barter section of the automat. Praying to RNGjesus and pressing the accept button would cause the automat to evaluate your offering and decide if its worthy of the chosen item.
